linera-core/src/remote_node.rs

```diff
@@ -198,7 +198,7 @@ impl<N: ValidatorNode> RemoteNode<N> {
         let mut found_blobs = if let Some(info) = info {
             let new_found_blobs = missing_blobs
                 .iter()
-                .filter_map(|blob_id| info.manager.pending_blobs.get(blob_id))
+                .filter_map(|blob_id| info.manager.locked_pending_blobs.get(blob_id))
```

Comment:
We should add to the method documentation, and maybe even the method _name_ that this only looks up the locked blocks.